src/test/java/com/celements/model/classes/fields/BooleanFieldTest.java

```diff
@@ -21,8 +21,8 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
```

Comment:
please do not override the default setUp method. Instead add a different named method with Before annotation.